Tests/Editor/ValidatorBlacklistedClassTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace DTValidator.TestBlacklisted
+{
+    public class BlacklistedOutletComponent : MonoBehaviour
+    {
+        public GameObject Outlet;
+    }
+}
+
+namespace DTValidator.Internal
+{
+    using DTValidator.TestBlacklisted;
+    using NUnit.Framework;
+
+    public static class ValidatorBlacklistedClassTests
+    {
+        private static IList<ValidatorBlacklistedClass> BlacklistedOutletComponentClassProvider()
+        {
+            var blacklistedClass = ScriptableObject.CreateInstance<ValidatorBlacklistedClass>();
+            blacklistedClass.Class = "DTValidator.TestIgnore";
```

Comment:
Should the class targeted be `BlacklistedOutletComponent`?

Reply:
Yes of course.